easybuild/easyconfigs/m/mapDamage/mapDamage-2.2.1_use-external-seqtk.patch

```diff
@@ -0,0 +1,74 @@
+Author: Jasper Grimm (UoY)
+Allow mapDamage to use a seqtk from $PATH, instead of looking in its install dir
```

Comment:
@jfgrimm To what extend did you test whether using an external `seqtk` with `mapDamage` is working correctly?

We got a problem report from a researcher trying to use the `mapDamage` module that was installed with the easyconfig file from #14725, which fails with:

```
Seqtk executable not accessible; mapDamage has not
been intalled properly or current user does not
sufficient permissions. Expected executable at
   '/software/mapDamage/2.2.1-foss-2021b/lib/python3.9/site-packages/mapdamage/seqtk/seqtk'
```

This patch would fix that, clearly, but apparently the `seqtk` that's included with `mapDamage` is a heavily patched/trimmed down version, compared to the original (`diff` produces a 2000-line patch on `seqtk.c`).

Reply:
as far as I understand, the `seqtk` they ship is simply a stripped down version, but a full `seqtk` should still be compatible. FWIW the conda recipe uses a patch to similar effect, and also uses an fullexternal `seqtk`, and that's listed as one of the recommended installation methods on mapDamage's homepage

Reply:
Excellent, thanks for clarifying!